lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -389,21 +433,6 @@ Ansible format, this function will convert the keys to snake_case.
 Converts a an Ansible list of filters to a boto3 friendly list of dicts.  This is useful for any
 boto3 `_facts` modules.
 
-### boto_exception
```

Comment:
Since we still have modules using boto and some of the exception handling probably could be improved in places I feel like this section shouldn't be deleted yet.

Reply:
Ah, I thought `boto_exception` was less widely used than it appears to be, I've removed this part of the change.